cron.sh

```diff
@@ -22,7 +22,11 @@ trap cleanup 1 2 3 6 9 15
 if [ ! -f $lockfile ];then
     echo $$ > $lockfile
 else
-    exit 0
+    if [ $(find $lockfile -mmin +59) ];then
+        rm -f $lockfile
+    else
+        exit 0
+    fi
```

Comment:
Careful! This removes the lockfile, but does not create a new one for the current cron run! Race conditions may occur.

Also, You can flatten the condition to avoid the nested if/else construct:

```bash
if [ ! -f $lockfile -o $(find $lockfile -mmin +59) ] ; then
    echo $$ > $lockfile
else
    exit 0
fi
```